scheduler/src/cook/components.clj

```diff
@@ -230,6 +231,16 @@
       (log/info "Found framework id in zookeeper:" framework-id)
       framework-id)))
 
+(defn auth-filter
```

Comment:
Should this be called `bypass-auth-filter` or `skip-auth-filter`. Otherwise, the name suggests to me that the filter is going to perform the authorization.

Reply:
Renamed to `conditional-auth-bypass`.